docs/index.md

```diff
@@ -9,24 +9,21 @@ button:
 
 features:
     title: About CAF Auditing
-    subtitle: The Audit service maintains a documented audit trail of user and system activity in a centralized database. The benefits of using the Audit service include traceability and accountability, but the audit records can also be used for analytical, archiving and reporting purposes.
+    subtitle:  The Audit service maintains a documented audit trail of user and system activity in a centralized database.  The benefits of using the Audit service include traceability and accountability, but the audit records can also be used for analytical, archiving and reporting purposes.
     feature_list:
         - image: assets/img/LandingIcons/support.png
           title: Support for Multiple Applications
-          subtitle: User and system audit events are defined per application. The Audit service offers an easy and user-friendly way to register these events using the Audit Management web service API.
+          subtitle: User and system audit events are defined per application.
         - image: assets/img/LandingIcons/multi-tenant.png
           title: Multi-Tenant Aware
-          subtitle: The service supports multiple tenants in a secure and scalable way. These tenants are registered using the Audit Management web service API. Multi-tenancy support also facilitates audit reporting on a per tenant basis.
+          subtitle: The service supports multiple tenants in a secure and scalable way.  Multi-tenancy support also facilitates audit reporting on a per tenant basis.
```

Comment:
Updated to: "The service supports multiple tenants in a secure and scalable way.  Multi-tenancy support also facilitates audit reporting on a per tenant basis and across multiple tenants"